tests/features/event.feature

```diff
@@ -281,12 +281,8 @@ Feature: Event content creation
     And I fill in "Content owner" with "Committee on Agriculture and Rural Development"
     And I fill in "Responsible department" with "Audit Board of the European Communities"
     And I fill in "Teaser" with "Event teaser"
-    And I press "Save"
-    Then I should see the following error messages:
-      | error messages                                                                                 |
-      | You have to fill in at least one of the following fields: Internal organiser or Organiser name |
     # Make sure that errors related to the Organiser fields are fixed.
-    When I check "Organiser is internal"
+    And I check "Organiser is internal"
```

Comment:
This was correct, it starts another assertion block.

Reply:
Fixed